sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -229,6 +237,8 @@ PollerFlux<AnalyzeActionsOperationDetail, AnalyzeActionsResultPagedFlux> beginAn
                 );
             }
 
+            throwIfTargetServiceVersionFoundForActions(this.serviceVersion,
+                Arrays.asList(TextAnalyticsServiceVersion.V3_0, TextAnalyticsServiceVersion.V3_1), actions);
```

Comment:
The logic of `throwIfTargetServiceVersionFoundForActions` is if found the API source version which the client is using in the target version list, we will throw an exception. So the target version is the version that the api/property should not call.
It won't be long I believe, since we only GA 2 GA versions in the past 3 years, and the older version might get deprecated at some time in the future when the service decided to not support it. It should be fine.